book/version_control.md

```diff
@@ -34,13 +34,27 @@ When used effectively, version control helps us to identify which changes have n
 
 Git is invaluable when recording and combining changes from multiple developers, as merging allows us to resolve conflicts between overlapping changes. Using a remote Git repository maintains a single source of truth, even when multiple individuals are working on a project. Additionally, version control platforms, like GitHub and GitLab, can make it easier to track and review ongoing work. This avoids duplication of effort and keeps review embedded in the development workflow.
 
-## Include or exclude
+## Include code and documentation in version control
 
-Ideally, you should include any code that is required to run your analysis. In a public repository, you may need to omit confidential or sensitive aspects of the project.
+Ideally, you should include all code and documentation that is required to run your analysis.
```

Comment:
This might be good to add, as we'll soon be adding guidance about assumptions and caveats etc
```suggestion
Ideally, you should include all code and documentation that is required to understand and run your analysis.
```